### ISSUE TITLE
Fork specification specific build-tools (protoc) into a specification-tools repository

### ISSUE THREAD

**Issue by jsuereth** [open-telemetry/opentelemetry-specification#3899]
Today the build-tools repository is an amalgamation of three major components:

- A docker image with C++ development tooling
- A docker image to validate the open-telemetry/opentelemetry-proto repository
- Tooling for semantic conventions.

Due to the split ownership model, no single group feels responsible and simple fixes like dependabot PRs have lagged in this repository.

I'd like to propose splitting out components with clear ownership.  For the Specification this means taking ownership of the proto related tooling and having a single repository that would be maintained by Specification owners (in addition to the opentelemetry-proto repository).

Related: https://github.com/open-telemetry/semantic-conventions/issues/767
Related: https://github.com/open-telemetry/opentelemetry-cpp/issues/2550

**Comment by reyang** [comment]
@lquerel FYI, this might be a good opportunity to have a joint effort with Weaver.

**Comment by lquerel** [comment]
> @lquerel FYI, this might be a good opportunity to have a joint effort with Weaver.

Yep. I'm collaborating with @jsuereth on this topic.

**Comment by ocelotl** [comment]
I asked Claude to review this issue and Claude suggests it be closed with this explanation:

> The proposal's goal was to give Specification owners clear ownership of proto-related tooling by splitting the amalgamated build-tools repo. This has happened: the two related split proposals (#767 semantic-conventions-tools, cpp #2550) are both closed, and open-telemetry/build-tools is now scoped to only Protobuf tooling ('* [Protobuf](./protobuf/README.md)') maintained by the OpenTelemetry Technical Committee / Specification Approvers. Confidence medium because the outcome was build-tools becoming proto-only rather than a literally-named new 'specification-tools' repo, but the clear-ownership intent is fulfilled.

To guard against a mistaken suggestion, Claude then independently re-verified this conclusion:

> Verified: sem-conv #767 and cpp #2550 both CLOSED/COMPLETED, and build-tools now contains only protobuf/schemas dirs (no cpp or semantic-conventions), scoped to Protobuf tooling under Specification Approvers — the body's ask (single spec-owned repo for proto tooling) is fulfilled despite the different repo name.

Claude evaluated this issue with medium confidence that it can be closed.

Claude has classified this issue as resolved.

The intention of this comment is to make it easier for a human being to review this issue before making a decision.

**Comment by jsuereth** [comment]
This has not been fixed - Claude is hallucinating.  Only some of the items have been fixed.  build-tools is still needed for the proto repository.

**Comment by jsuereth** [comment]
> To guard against a mistaken suggestion, Claude then independently re-verified this conclusion:
>
> Verified: sem-conv https://github.com/open-telemetry/opentelemetry-specification/pull/767 and cpp https://github.com/open-telemetry/opentelemetry-specification/pull/2550 both CLOSED/COMPLETED, and build-tools now contains only protobuf/schemas dirs (no cpp or semantic-conventions), scoped to Protobuf tooling under Specification Approvers — the body's ask (single spec-owned repo for proto tooling) is fulfilled despite the different repo name.

@ocelotl You still need to verify these results yourself or farm out the task.  The CPP link is for a  Java semantic convention change that has nothing to do with this issue.

**Comment by ocelotl** [comment]
@jsuereth yes, as mentioned in my comment above, the idea here is to assist humans in reviewing this issue, not to just let AI decide.
